index.html

```diff
@@ -0,0 +1,59 @@
+<!DOCTYPE html>
+
+<html>
+
+<head>
+<title>My First Website</title>
+<meta charset="UTF-8"/>
+<meta name="keywords" content="HTML" />
+<meta name="description" content="My First Website"/>
+<meta name="author" content="Ipek Kaplan" />
+<!---meta etiketleri ayfanın tarayıcılarda ve arama sonuçlarında nasıl görüneceğini 
+    belirlemede ve arama motorlarının sayfayı anlamasında ve daha karışıklık gidermesinde etkilidir.-->
+</head>
+
+<body style="background-color: aquamarine;">
+<h1>Hos Geldiniz!</h1>
+<nav>
+    <ul>
+     <li> <a href="anasayfa.html" target="_blank">Anasayfa Dokumani</a><br></li>
+      <li> <a href="hakkımızda.html" target="_blank">Hakkimizda Dokumani</a><br></li>
+       <li>   <a href="iletisim.html"target=_blank>Iletisim Dokumani</a><br></li>
+    </ul>
+    </nav>
+    <!--nav etiketi navigasyon ifadesinin kıaltmasıdır.
+    Kulanıcının sayfa içerisinde web adresleri arasında geçiş yapmasını sağlar.-->
+
+    <h1>Favori Yemeklerim</h1>
+    <ol>
+        <li>Köfte</li>
+        <li>Pizza</li>
+        <li>Salata</li>
+    </ol>
+    <!--burada ordered(sıralı)liste kullanılmıştır.-->
+
+    <h1>Resim Ekleme</h1>
+<img src="./vsco_102422(7).jpg"

```

Comment:
```suggestion
<img src="https://img.freepik.com/free-photo/painting-mountain-lake-with-mountain-background_188544-9126.jpg?size=626&ext=jpg&ga=GA1.1.2113030492.1708992000&semt=sph"
```

Localde tuttuğun dosyalar için doğru bir kaynak gösterme fakat taskta internet üzerinden kanyak gösterme istemiştik. Yukarıdaki gibi bir düzenleme yapabilirsin.